### PR TITLE
Allows PAI range restrictions to be toggled on/off

### PR DESCRIPTION
**Pull request by vinylspiders**
## About The Pull Request

New setting to enable or disable the 'leash' on the PAI. It starts disabled, allowing PAI's to roam freely by default as they were previously able to. The leash can be activated at will and will yoink the holoform back to you once enabled if the PAI is outside its specified range.

## How This Contributes To The Skyrat Roleplay Experience

Leaves whether or not to leash their PAIs up to the player.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

firefox_48qvNvriKQ

</details>

## Changelog

:cl:
qol: made whether or not to leash your PAI a toggle in the PAI card menu, off by default. the leash can be toggled on or off at will, giving an option to allow PAIs to roam freely or stick within a specified range.
qol: also bumps the max range for the leash up to 25 (from 9 previously)
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/pai/card.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+	.["pai"]["leash_enabled"] = pai.leashed
+
+/obj/item/pai_card/ui_act(action, list/params, datum/tgui/ui)
+	. = ..()
+	if(.)
+		return TRUE
+	if(pai && action == "toggle_leash")
+		pai.toggle_leash()
+		return TRUE
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/pai/card.dm:15]:
```suggestion

	if(pai && action == "toggle_leash")
		pai.toggle_leash()
		return TRUE

```

File: modular_skyrat/master_files/code/modules/pai/pai.dm [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/mob/living/silicon/pai/
+	/// Whether or not the PAI is subject to range limitations and able to roam freely, off by default
+	var/leashed = FALSE
+
+/// Enables or disables the leash, allowing or forbidding the PAI from leaving a specified range
+/mob/living/silicon/pai/proc/toggle_leash()
+	leashed = !leashed
+	to_chat(src, span_warning("Your leash has been [leashed ? "activated" : "deactivated"]!"))
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/pai/pai.dm]:
```suggestion
	to_chat(src, span_warning("Your virtual leash has been [leashed ? "activated" : "deactivated"]!"))
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Minor nitpicks.

**Review by Gandalf2k15 — APPROVED** [review]
Ok

**Pull request merged**